tests/src/main/java/org/hibernate/beanvalidation/tck/tests/xmlconfiguration/constraintdeclaration/MandatoryNameAttributeTest.java

```diff
@@ -35,15 +38,9 @@ public static WebArchive createTestArchive() {
 				.build();
 	}
 
-	@Test
+	@Test(expectedExceptions = ValidationException.class)
 	@SpecAssertion(section = Sections.XML_MAPPING_CONSTRAINTDECLARATIONINXML_CONSTRAINTDECLARATION, id = "b")
 	public void testNameAttributeIsMandatory() {
-		try {
-			TestUtil.getValidatorUnderTest();
-			fail();
-		}
-		catch ( ValidationException e ) {
-			// success
-		}
+		TestUtil.getValidatorUnderTest();
```

Comment:
Same question here.

Reply:
Looking further, it seems to be handled a bit inconsistently. I don't think the `fail()` calls would be needed, would they?